.github/workflows/test_linux_cuda.yaml

```diff
@@ -0,0 +1,68 @@
+
+name: Unit-tests on Linux GPU
+
+on:
+  pull_request:
+  push:
+    branches:
+      - nightly
+      - main
+      - release/*
+  workflow_dispatch:
+
+jobs:
+  tests:
+    strategy:
+      matrix:
+        python_version: ["3.8", "3.9", "3.10"]
```

Comment:
We shouldn't test on 3.8 as we don't support it.
For now, a single version should be enough on this job, so let's drop 3.10 too.